src/services/packageService.test.ts

```diff
@@ -181,12 +182,13 @@ describe("Package Service", () => {
     await packageService.createBinding(functionName, functionMetadata);
 
     expect(mkdirSpy).toBeCalledWith(expectedFolderPath);
-    const call = writeFileSpy.mock.calls[0];
+    mkdirSpy.mockRestore();
```

Comment:
why restoring these here instead of at the end like before?

Reply:
We saw this a bunch in VoTT as well. For some reason, if you don't restore some spies individually, the tests conflict with each other. This makes it more explicit and guarantees that they will be restored.